demos/classification_demo/python/classification_demo.py

```diff
@@ -39,71 +39,87 @@
-                                 'mode (for HETERO and MULTI device cases use format '
-                                 '<device1>:<nstreams1>,<device2>:<nstreams2> or just <nstreams>).',
-                            default='', type=str)
-    infer_args.add_argument('-nthreads', '--num_threads', default=None, type=int,
-                            help='Optional. Number of threads to use for inference on CPU (including HETERO cases).')
+
+    infer_args.add_argument('-nireq', '--num_infer_requests', default=0, type=int, metavar="<NUMBER>",
+        help='number of infer requests')
+
+    infer_args.add_argument('-nstreams', '--num_streams', default='', type=str, metavar="<NUMBER>",
+        help='number of streams to use for inference on the CPU or/and GPU in throughput '
+            'mode (for HETERO and MULTI device cases use format '
+            '<device1>:<nstreams1>,<device2>:<nstreams2> or just <nstreams>)')
+
+    infer_args.add_argument('-nthreads', '--num_threads', default=None, type=int, metavar="<NUMBER>",
+        help='number of threads to use for inference on CPU (including HETERO cases)')
 
     io_args = parser.add_argument_group('Input/output options')
+
     io_args.add_argument('--loop', default=False, action='store_true',
-                         help='Optional. Enable reading the input in a loop.')
-    io_args.add_argument('-o', '--output', required=False,
-                         help='Optional. Name of the output file(s) to save.')
-    io_args.add_argument('-limit', '--output_limit', required=False, default=1000, type=int,
-                         help='Optional. Number of frames to store in output. '
-                              'If 0 is set, all frames are stored.')
-    io_args.add_argument('--no_show', help="Optional. Don't show output.", action='store_true')
-    io_args.add_argument('--output_resolution', default=None, type=resolution,
-                         help='Optional. Specify the maximum output window resolution '
-                              'in (width x height) format. Example: 1280x720. '
-                              'Input frame size used by default.')
-    io_args.add_argument('-u', '--utilization_monitors', default='', type=str,
-                         help='Optional. List of monitors to show initially.')
+        help='enable reading the input in a loop')
+
+    io_args.add_argument('-o', '--output', required=False, metavar="<OUTPUT>",
+        help='name of the output file(s) to save')
+
+    io_args.add_argument('-limit', '--output_limit', required=False, default=1000, type=int, metavar="<NUMBER>",
+        help='number of frames to store in output. If 0 is set, all frames are stored. Default value is 1000')
+
+    io_args.add_argument('--no_show', action='store_true',
+        help="don't show output")
+
+    io_args.add_argument('--output_resolution', default=None, type=resolution, metavar="<STRING>",
```

Comment:
This is equivalent to `--res` from classification_benchmark_demo. With just different default value. Please align them except for the default value.

Reply:
Done

Reply:
@ieliz please, could you also restore the `type=resolution`?